<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>710</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Настройка интервалов</string>
  </property>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>От</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="fromInterval_timeEdit">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>H.m.s.zzz</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>До</string>
   </property>
  </widget>
  <widget class="QTimeEdit" name="toInterval_timeEdit">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="displayFormat">
    <string>H.m.s.zzz</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="alternativeIntervals_groupBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>240</y>
     <width>230</width>
     <height>150</height>
    </rect>
   </property>
   <property name="title">
    <string>Альтернативный тип</string>
   </property>
   <widget class="QPushButton" name="addAllIntervals_pushButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>100</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить все</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>140</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Выберите интервал</string>
    </property>
   </widget>
   <widget class="QComboBox" name="alternativeIntervals_comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>210</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContents</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="addInterval_pushButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>97</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
  </widget>
  <widget class="QListWidget" name="intervals_listBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>89</y>
     <width>450</width>
     <height>251</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="deleteInterval_button">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>350</y>
     <width>110</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Удалить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="clearAllIntervals_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>350</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Очистить все</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="joinIntervals_groupBox">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>230</width>
     <height>170</height>
    </rect>
   </property>
   <property name="title">
    <string>Склеить интервалы</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>80</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Интервал 1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>80</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Интервал 2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="joinIntervals_pushButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>130</y>
      <width>90</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Склеить</string>
    </property>
   </widget>
   <widget class="QComboBox" name="interval1_comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>210</width>
      <height>23</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContents</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="interval2_comboBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>210</width>
      <height>23</height>
     </rect>
    </property>
    <property name="maxVisibleItems">
     <number>10</number>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QComboBox::AdjustToContents</enum>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="saveIntervals_button">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>350</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="changeInterval_button">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addInterval_button">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>120</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>450</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
